public/components/trace_analytics/components/dashboard/dashboard_content.tsx

```diff
@@ -77,7 +79,12 @@ export function DashboardContent(props: DashboardProps) {
   }, [showTimeoutToast]);
 
   useEffect(() => {
-    chrome.setBreadcrumbs([parentBreadcrumb, ...childBreadcrumbs]);
+    if (isNavGroupEnabled) {
+      chrome.setBreadcrumbs([...childBreadcrumbs]);
+    } else {
+      chrome.setBreadcrumbs([parentBreadcrumb, ...childBreadcrumbs]);
+    }
+
```

Comment:
Can we try using 
```
useEffect(() => {
       setNavBreadCrumbs([], [parentBreadcrumb, ...childBreadcrumbs]);
    }, [isNavGroupEnabled])
```

Reply:
Does not work unfortunately. 
<img width="673" alt="Screenshot 2024-07-23 at 2 59 58 PM" src="https://github.com/user-attachments/assets/b4b53e08-16c7-448c-912e-117666327d0d">

Reply:
In that case, can we please track this in an issue. Cause ideally we would want to keep functions re-used across components.